spec/lib/hosting/hetzner_apis_spec.rb

```diff
@@ -38,6 +38,25 @@
     end
   end
 
+  describe "reset" do
+    it "can reset a server" do
+      expect(Config).to receive(:hetzner_ssh_key).and_return("ssh-rsa TestKey").at_least(:once)
```

Comment:
can probably factor this into a `before`

Reply:
saw the last one, which returns nil for the hetzner key, nvm

Reply:
aha we found that the ssh key is a holdover from the reimage thing, can delete stuff (and @0xjunhao did)